src/index.js

```diff
@@ -29,6 +30,7 @@ function render(ui, {container, baseElement = container, queries} = {}) {
       // Intentionally do not return anything to avoid unnecessarily complicating the API.
       // folks can use all the same utilities we return in the first place that are bound to the container
     },
+    asFragment: () => JSDOM.fragment(container.innerHTML),
```

Comment:
Is there a reason to do this with JSDOM rather than:

```javascript
const fragment = document.createDocumentFragment()
fragment.innerHTML = container.innerHTML
```

That way we don't have to rely on JSDOM directly. I want this package to be capable of running in the browser.

Reply:
Great point! Pushed a new commit using plain DOM APIs for it.

Followed https://stackoverflow.com/a/25214113/1850276, but JSDOM doesn't support Ranges (https://github.com/jsdom/jsdom/issues/317), so I added a fallback for it